lib/app/list.ex

```diff
@@ -0,0 +1,53 @@
+defmodule App.List do
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query
+  alias App.{Item, ListItem, Person, Repo}
+  alias __MODULE__
+
+  schema "lists" do
+    field :name, :string
+
+    belongs_to :people, Person, references: :person_id, foreign_key: :person_id
+    many_to_many(:items, Item, join_through: ListItem)
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(list, attrs \\ %{}) do
+    list
+    |> cast(attrs, [:person_id, :name])
+    |> validate_required([:person_id, :name])
+    |> unique_constraint([:name, :person_id], name: :lists_name_person_id_index)
+  end
+
+  def create_list(attrs) do
+    %List{}
+    |> changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def get_list!(id), do: Repo.get!(List, id)
+
+  def get_lists_from_ids(ids) do
+    Repo.all(from l in List, where: l.id in ^ids)
+  end
+
+  def list_person_lists(person_id) do
```

Comment:
`list_person_lists/1` is pretty much the definition of a confusing function name. 😜 
would much sooner the function was just: `get_lists(group_id)` 
As noted in https://github.com/dwyl/auth/issues/220#issuecomment-1273033055
there isn't much point to making `lists` belong to a `person` as the objective is very much to enable sharing/collaboration ASAP. 🤞